agent/credentials/manager.go

```diff
@@ -48,6 +48,12 @@ type IAMRoleCredentials struct {
 	Expiration string `json:"Expiration"`
 }
 
+// TaskIAMRoleCredentials wraps the task arn and the credentials object for the same
+type TaskIAMRoleCredentials struct {
+	Arn                string
```

Comment:
I think this should be `ARN`: https://github.com/golang/go/wiki/CodeReviewComments#initialisms